js/templates/modals/moderatorDetails.html

```diff
@@ -31,14 +33,14 @@
     <div class="contentBox mDetailBox flexRow flexVCent gutterH padMd clrP clrBr clrSh3">
       <div class="flexExpand">
         <div class="flexCol flexCent">
-          <div>📍 <%= ob.location || ob.polyT('userPage.noLocation') %></div>
+          <div><%= ob.parseEmojis('📍') %> <%= ob.location || ob.polyT('userPage.noLocation') %></div>
           <div class="tx5b clrT2"><%= ob.polyT('moderatorDetails.location') %></div>
         </div>
       </div>
-      <div class="rowDivV clrBr"></div>
-      <div class="flexExpand">
+      <div class="rowDivV clrBr TODO"></div>
+      <div class="flexExpand TODO">
         <div class="flexCol flexCent">
-          👍 XX<% // placeholder for reputation %>
+          <%= ob.parseEmojis('👍') %> XX<% // placeholder for reputation %>
```

Comment:
This is hidden with the todo class, but I added the parse emoji anyways so it's not forgotten in the future.